src/util.cpp

```diff
@@ -1446,11 +1448,20 @@ string FormatFullVersion()
 
 #endif
 
+double Round(double d, int place)
+{
+    const double accuracy = place != 0
+            ? std::pow(10, place)
+            : 1;
+
+    return std::round(d * accuracy) / accuracy;
+}
+
 std::string RoundToString(double d, int place)
 {
     std::ostringstream ss;
-    ss << std::fixed << std::setprecision(place) << d ;
-    return ss.str() ;
+    ss << std::fixed << std::setprecision(place) << d;
```

Comment:
Aha, this is just a whitespace change.
Once changing [the above](#discussion_r132849848), I approve.